arch/arm/boot/dts/overlays/vc4-kms-dsi-7inch-overlay.dts

```diff
@@ -6,51 +6,109 @@
 /dts-v1/;
 /plugin/;
 
+#include "edt-ft5406.dtsi"
+
 / {
 	compatible = "brcm,bcm2835";
```

Comment:
The compatible string is inherited from edt-ft5406.dtsi, soi this is superfluous.

Reply:
Done. Comment inserted to make it obvious as to why.